tests/test_rio_blocks.py

```diff
@@ -0,0 +1,132 @@
+"""Unittests for $ rio blocks"""
+
+
+import itertools as it
+import json
+import os
+import re
+
+import numpy as np
+import six
```

Comment:
@geowurster `six` isn't declared in `requirements-dev.txt` and isn't otherwise used in rasterio.  I got the impression we were actively avoiding it in favor of the lighter weight `compat.py` in rasterio.  It has `zip_longest`.